components/raftstore/src/store/util.rs

```diff
@@ -349,9 +352,16 @@ pub fn check_flashback_state(
     }
     let is_flashback_request = WriteBatchFlags::from_bits_truncate(req.get_header().get_flags())
         .contains(WriteBatchFlags::FLASHBACK);
-    // If the region is in the flashback state, the only allowed request is the
-    // flashback request itself.
+    // If the region is in the flashback state:
+    //   - A request with flashback flag will be allowed.
+    //   - A read request whose `read_ts` is smaller than `flashback_start_ts` will
+    //     be allowed.
     if is_in_flashback && !is_flashback_request {
```

Comment:
Should we change `is_in_flashback` to `(is_in_flashback || flashback_start_ts > 0)` because the latest proto says we need to only check `flashback_start_ts`, also in case we remove `is_in_flashback` in the future?